vg/vg.go

```diff
@@ -85,6 +88,49 @@ type CanvasSizer interface {
 	Size() (x, y Length)
 }
 
+// CanvasWriterTo wraps behavior required for writing
+// a Canvas representation to a concrete image.
+type CanvasWriterTo interface {
+	CanvasSizer
+	io.WriterTo
+}
+
+var (
+	formatLock sync.RWMutex
+	formats    = make(map[string]func(w, h Length) CanvasWriterTo)
```

Comment:
no, I don't think it's necessary.
it's customary (at least in the tools I have dealt with so far) to just generate a title (say, "gonum/plot-x11 [counter]") for these nameless windows.